backend/amd64/emit.mlp

```diff
@@ -1383,7 +1391,7 @@ let emit_probe_handler_wrapper p =
   cfi_endproc ();
   emit_function_type_and_size wrap_label
 
-let emit_probe_notes0 () =
+let emit_probe_notes0 probes =
   begin match system with
   | S_gnu | S_cygwin | S_solaris | S_win32 | S_linux_elf | S_bsd_elf
   | S_beos | S_mingw | S_win64 | S_linux | S_mingw64 | S_unknown ->
```

Comment:
(Not part of this pull request, but this seems a bit
wide; shouldn't _e.g._ Windows and unknown have
their own case -- doing nothing? If changed, beware
of other occurences.)

Reply:
At configure time, probes are disabled on all windows-based systems, so the corresponding cases won't be reachable.